common/djangoapps/util/password_policy_validators.py

```diff
@@ -92,6 +100,10 @@ def validate_password_complexity(value):
         errors.append(_("must contain {0} or more non ascii characters").format(complexities["NON ASCII"]))
     if len(words) < complexities.get("WORDS", 0):
         errors.append(_("must contain {0} or more unique words").format(complexities["WORDS"]))
+    if len(numeric) < complexities.get("NUMERIC", 0):
+        errors.append(_("must contain {0} or more numeric characters").format(complexities["NUMERIC"]))
```

Comment:
For the user strings, can we use "or more numbers" and "or more letters"? Probably easier for folks to understand.